sensor.py

```diff
@@ -63,7 +63,13 @@ def setup_platform(
         GridImportSensor(cache),
         GridExportSensor(cache),
         SolarEnergySensor(cache),
-        GeneratorUseSensor(cache),
```

Comment:
Not a typo, this is duplicated from a few lines above

Reply:
Amazing, thankyou 😅